lib/_stream_readable.js

```diff
@@ -458,6 +458,12 @@ Readable.prototype.read = function(n) {
       endReadable(this);
   }
 
+  // Drop data if '_dumped' flag exists. This is for IncomingMessage.
```

Comment:
Sorry to be nit pick here, but maybe `http.IncomingMessage` just to make it clear that it's `http`-specific?